src/check-response-schema.ts

```diff
@@ -10,12 +10,13 @@ use(chaiPlugin({ apiDefinitionsPath }))
 /**
  * Check that the response matches the API schema.
  *
- * Ignores /v1 prefix
+ * Ignores /vX prefix
  */
 export function checkResponseSchema(response: AxiosResponse) {
-  if (response.request.path.slice(0, 3) === '/v1') {
-    // removes /v1 prefix, total hack to get api schema matcher to work
-    response.request.path = response.request.path.slice(3)
+  const versionPrefixMatch = response.request.path.match(/\/v([0-9]+)/)
```

Comment:
should we do 
```suggestion
  const versionPrefixMatch = response.request.path.match(/^\/v([0-9]+)/)
```
so only the prefix is matched and not anywhere in the string?